GUI/add_new.py

```diff
@@ -44,7 +46,7 @@ def add_new_game(json_to_add, is_new):
 	"""
 	if "" in json_to_add.items():
 		raise DatabaseError("Some fields are left void. Please complete them and try again.")
-	with open('games.json', 'r') as jsonfile:
+	with urlopen('http://dogeek.legtux.org/games.json') as jsonfile:
```

Comment:
Loading stuff over http is a bad idea, tons of ISPs like to inject things into pages without checking that the page is HTML first.

Reply:
> Well, can't connect to the repo through FTP,

So why not just use http?

> Apparently, there's a library called gitpython that allows you to interact with repositories, though it needs to have git installed

Why not just load the raw json from github over http?

> Plus, You don't have to worry about the domain not getting renewed/the server going offline, it's my own FTP

It still might go down. You could lose interest in the project or get hit by a bus or OVH could decide that this is "adult content" and ban your account. Github is much more reliable

Reply:
the thing is, you can read but you can't write, that doesn't seem like a good idea to use github then. Maybe we should just keep it local then, but the database can't be synchronized over the web.

Reply:
> the thing is, you can read but you can't write, that doesn't seem like a good idea to use github then

Why would we write?

Reply:
How else are we supposed to update the database, and let user add new games to it ?

Reply:
I've changed my password since it was in clear (I'm a dumbass), so no worry, nothing has been compromised. I've registered a free account at gearhost, which offers free mysql servers, so the best  thing would be to port the database to MySQL